src/api/visualization.js

```diff
@@ -0,0 +1,101 @@
+import { getDimensionMetadataFields } from '../modules/visualization.js'
+
+const dimensionFields = () =>
+    'dimension,dimensionType,filter,program[id],programStage[id],optionSet[id],valueType,legendSet[id],repetition,items[dimensionItem~rename(id)]'
+
+export const VISUALIZATION_QUERY = {
+    eventVisualization: {
+        resource: 'eventVisualizations',
+        id: ({ id }) => id,
+        // TODO: check if this list is what we need/want (copied from old ER)
+        params: ({ nameProp, withSubscribers }) => ({
+            fields: [
+                '*',
+                `columns[${dimensionFields}]`,
+                `rows[${dimensionFields}]`,
+                `filters[${dimensionFields}]`,
+                `program[id,programType,${nameProp}~rename(name),displayEnrollmentDateLabel,displayIncidentDateLabel,displayIncidentDate,programStages[id,displayName~rename(name),repeatable]]`,
+                'programStage[id,displayName~rename(name),displayExecutionDateLabel,displayDueDateLabel,hideDueDate,repeatable]',
+                `programDimensions[id,${nameProp}~rename(name),enrollmentDateLabel,incidentDateLabel,programType,displayIncidentDate,displayEnrollmentDateLabel,displayIncidentDateLabel,programStages[id,${nameProp}~rename(name),repeatable,hideDueDate,displayExecutionDateLabel,displayDueDateLabel]]`,
+                'access',
+                'href',
+                !withSubscribers && '!subscribers',
```

Comment:
This ends up adding 2 `false` entries in the list of fields:
<img width="544" alt="Screenshot 2025-05-22 at 16 38 00" src="https://github.com/user-attachments/assets/501253d0-77b2-46dc-bb6e-b47f5a26ff47" />